### ISSUE TITLE
What's the status of the meson build?

### ISSUE THREAD

**Issue by nacho** [centricular/cairo#1]
same questions as for fontconfig, in gvsbuild we already had to extend the nmake build system in order to build gobject so I would definitely be interested on this

**Comment by ao2** [comment]
Just a heads up that meson build files have been proposed to upstream cairo in https://gitlab.freedesktop.org/cairo/cairo/-/merge_requests/37

**Comment by MathieuDuponchelle** [comment]
@ao2 yeah I know :P Hadn't noticed your review however, sorry, will look later :)

**Comment by ao2** [comment]
> @ao2 yeah I know :P Hadn't noticed your review however, sorry, will look later :)

Thank you @MathieuDuponchelle, my message was meant as a reply to @nacho but it was nice that you got it too :)

**Comment by MathieuDuponchelle** [comment]
Oh my bad, saw the mail and replied without paying attention to the context
